test/test_torch.py

```diff
@@ -3319,6 +3319,30 @@ def rand_tensor(size, dtype, device):
             out = torch.addcmul(a, b, c, value=-1)
             self.assertTrue(not (out.isnan() or out.isinf()))
 
+    def test_addcmul_cpu_scalar_errors(self, device):
+        # Logic is dtype agnostic, so dtype isn't tested
```

Comment:
huh is this the way to not have dtype be parametrized? Could you look through the logs and check that this test isn't just running the same thing 8 times anyway?

Reply:
It only ran 2 times because I was doing "print(device)" while debugging -- I assumed it would work because the other tests had a @dtypes but not @devices